packages/csshnpd/Makefile

```diff
@@ -19,8 +19,9 @@ include $(INCLUDE_DIR)/cmake.mk
 CMAKE_OPTIONS += \
 		-S . \
 		-DBUILD_SHARED_LIBS=off \
-		-DCMAKE_C_COMPILER=gcc \
-		-DCMAKE_C_FLAGS="-Wno-calloc-transposed-args"
+		-DBUILD_TESTS=off \
+		-DCMAKE_C_COMPILER=$(TARGET_CC) \
```

Comment:
@XavierChanth this should be the most generic solution to selecting the appropriate (cross) compiler